pkg/cmd/alias/set/set.go

```diff
@@ -43,21 +43,28 @@ func NewCmdSet(f *cmdutil.Factory, runF func(*SetOptions) error) *cobra.Command
 			for commands with mixed quotes or multiple lines.
 
 			If '--shell' is specified, the alias will be run through a shell interpreter (sh). This allows you
-			to compose commands with "|" or redirect with ">". Note that extra arguments following the alias
+			to compose commands with '|' or redirect with '>'. Note that extra arguments following the alias
```

Comment:
Why this change?

Reply:
I tried explaining it in the commit message for 23ffca45f7dd7042239e7b89f5907a5f1883324d, but basically I changed it for consistency.  The first paragraph uses single quotes to refer to things you'd type in the shell, but then later switches to double quotes to do the same thing.

Also, since this doc string is embedded in Go source code, I thought it'd be easier to use single quotes rather than double quotes.